### PR TITLE
Add feed.documentation

### PR DESCRIPTION
**Pull request by wagner-certat**
It's a string, not strictly an URL to give more freedom to the user

fixes certtools/intelmq#749

@sykaeh please also review (can't select you for the ticket)

### CODE REVIEW

File: docs/Harmonization-fields.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Harmonization field names
 ||extra|[JSON](#json)|All anecdotal information, which cannot be parsed into the data harmonization elements. E.g. os.name, os.version, etc.  **Note**: this is only intended for mapping any fields which can not map naturally into the data harmonization. It is not intended for extending the data harmonization with your own fields.|
 |Feed|feed.accuracy|[Accuracy](#accuracy)|A float between 0 and 100 that represents how accurate the data in the feed is|
 |Feed|feed.code|[String](#string)|Code name for the feed, e.g. DFGS, HSDAG etc.|
+|Feed|feed.documentation|[String](#string)|Upstream's documentation of this feed.|
```
Comment (dmth) [inline docs/Harmonization-fields.md]: What is Upstream? I suggest the formulation:
"A URL or hint where to find the documentation of this feed"

File: intelmq/etc/harmonization.conf [hunk 2]
```diff
@@ -141,6 +141,10 @@
             "length": 100,
             "type": "String"
         },
+        "feed.documentation": {
+            "description": "Upstream's documentation of this feed.",
```
Comment (dmth) [inline intelmq/etc/harmonization.conf]: see harmonization-fields.md

File: intelmq/etc/harmonization.conf [hunk 3]
```diff
@@ -346,6 +350,10 @@
...
```
Comment (dmth) [inline intelmq/etc/harmonization.conf]: see harmonization-fields.md

Comment (SYNchroACK) [inline intelmq/etc/harmonization.conf]: see harmonization-fields.md

**Comment by SYNchroACK** [comment]
> Thank you for adding this to the DHO.
> 
> In addition to your changes, there should be a note stating the DHO has changed and additional columns will be required in the eventDB if the postgresql output bot is used. Maybe you can even provide the SQL-snippet which creates this column.
> 
> BR
> Dustin

@dmth I saw not that there is a README.md for postgresql bot and also a script [here](https://github.com/certtools/intelmq/blob/master/intelmq/bin/intelmq_psql_initdb.py) to generate the table. Is this suits the requirements that you mentioned?

**Comment by wagner-certat** [comment]
This does not alter existing tables. However, we have never done this before and therefore there's no existing place to store migrations.

**Comment by SYNchroACK** [comment]
@dmth do you still see any issue here?

**Comment by dmth** [comment]
I think we should provide hints for migrations, when the DHO has changed.

For parser-Configurations we did this here: https://github.com/certtools/intelmq/pull/866

In our Integration branch, we are collecting "Release-Notes", this might be a place to store these hints for an IntelMQ Administrator

https://github.com/Intevation/intelmq/blob/integrated/debian/release-notes.txt

**Comment by wagner-certat** [comment]
Use this to update the table:
```sql
ALTER TABLE events
   ADD COLUMN "feed.documentation" text;
```

**Pull request merged**